packages/omgx/wallet/deploy/001-ERC20Test.deploy.ts

```diff
@@ -0,0 +1,110 @@
+/* Imports: External */
+import { DeployFunction, DeploymentSubmission } from 'hardhat-deploy/dist/types'
+import { Contract, ContractFactory, utils} from 'ethers'
+import chalk from 'chalk';
+
+import L1ERC20Json from '../artifacts/contracts/L1ERC20.sol/L1ERC20.json'
+import L1ERC20GatewayJson from '../artifacts/contracts/L1ERC20Gateway.sol/L1ERC20Gateway.json'
+import L2DepositedERC20Json from '../artifacts-ovm/contracts/L2DepositedERC20.sol/L2DepositedERC20.json'
+
+let Factory__L1ERC20: ContractFactory
+let Factory__L2DepositedERC20: ContractFactory
+let Factory__L1ERC20Gateway: ContractFactory
+
+let L1ERC20: Contract
+let L2DepositedERC20: Contract
+let L1ERC20Gateway: Contract
+
+
+//Test ERC20
+const initialSupply = utils.parseEther("10000000000")
+const tokenName = 'JLKN'
+const tokenSymbol = 'JLKN'
+
+const deployFn: DeployFunction = async (hre) => {
+    // If TEST env var is not undefined we deploy these test contractgs
+    if (process.env.TEST) {
```

Comment:
i was thinking, we could avoid depl scripts for test contracts, but this directly combines the tokenPool, so not test and we should keep it. 
should we change the env var to imply if someone wanted to deploy tokenPool with a new token instead of test

Reply:
I'll remove this script completely and let the test deploy things it needs.

Reply:
but let me do this in the subsequent PR that enables wallet integration tests!